internal/oci/uvm.go

```diff
@@ -79,7 +81,11 @@ const (
 	// `spec.Windows.Resources.Storage.Iops`.
 	AnnotationContainerStorageQoSIopsMaximum = "io.microsoft.container.storage.qos.iopsmaximum"
 	// AnnotationGPUVHDPath overrides the default path to search for the gpu vhd
-	AnnotationGPUVHDPath            = "io.microsoft.lcow.gpuvhdpath"
+	AnnotationGPUVHDPath                  = "io.microsoft.lcow.gpuvhdpath"
+	AnnotationAssignedDeviceKernelDrivers = "io.microsoft.assigneddevice.kerneldrivers"
+	// AnnotationAssignedDeviceUtilityTool overrides the default path to search for device-util
+	AnnotationAssignedDeviceUtilityTool = "io.microsoft.assigneddevice.utilitytool"
```

Comment:
I think we should just remove this annotation entirely. I can't think of a case where a customer would want to customize where we get an internal tool from, and if they really need to, they can do so via the containerd config.

Reply:
we have no other way to setup a custom path without the annotation, since we need to put the value on the container spec somehow for hcsoci to use.